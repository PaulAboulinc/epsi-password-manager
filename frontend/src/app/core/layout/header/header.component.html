<mat-toolbar color="primary">
  <span>Recipe Book</span>
  <span class="example-spacer"></span>
  <button mat-icon-button class="example-icon favorite-icon" aria-label="Example icon-button with heart icon">
    <span class="material-icons">cake</span>
  </button>
  <button mat-icon-button class="example-icon" aria-label="Example icon-button with menu icon">
    <mat-icon>import_contacts</mat-icon>
  </button>
</mat-toolbar>
