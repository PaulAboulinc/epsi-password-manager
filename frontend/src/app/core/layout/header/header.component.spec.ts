import { ComponentFixture, TestBed } from '@angular/core/testing';

import { HeaderComponent } from './header.component';
import {MaterialModule} from "../../../shared/material.module";
import { BrowserAnimationsModule } from '@angular/platform-browser/animations';

describe('HeaderComponent', () => {
  let component: HeaderComponent;
  let fixture: ComponentFixture<HeaderComponent>;

  beforeEach(async () => {
    await TestBed.configureTestingModule({
      imports: [MaterialModule, BrowserAnimationsModule],
      declarations: [ HeaderComponent ]
    })
    .compileComponents();
  });

  beforeEach(() => {
    fixture = TestBed.createComponent(HeaderComponent);
    component = fixture.componentInstance;
    fixture.detectChanges();
  });

  it('should create', () => {
    expect(component).toBeTruthy();
  });

  it('should equal snapshot', () => {
    expect(fixture).toMatchSnapshot();
  });
});
