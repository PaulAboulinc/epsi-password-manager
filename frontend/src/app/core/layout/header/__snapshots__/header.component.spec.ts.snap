// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderComponent should equal snapshot 1`] = `
<app-header>
  <mat-toolbar
    class="mat-toolbar mat-primary mat-toolbar-single-row"
    color="primary"
  >
    <span>
      Recipe Book
    </span>
    <span
      class="example-spacer"
    />
    <button
      aria-label="Example icon-button with heart icon"
      class="example-icon favorite-icon mat-focus-indicator mat-icon-button mat-button-base"
      mat-icon-button=""
    >
      <span
        class="mat-button-wrapper"
      >
        <span
          class="material-icons"
        >
          cake
        </span>
      </span>
      <span
        class="mat-button-ripple mat-ripple mat-button-ripple-round"
        matripple=""
      />
      <span
        class="mat-button-focus-overlay"
      />
    </button>
    <button
      aria-label="Example icon-button with menu icon"
      class="example-icon mat-focus-indicator mat-icon-button mat-button-base"
      mat-icon-button=""
    >
      <span
        class="mat-button-wrapper"
      >
        <mat-icon
          aria-hidden="true"
          class="mat-icon notranslate material-icons mat-icon-no-color"
          data-mat-icon-type="font"
          role="img"
        >
          import_contacts
        </mat-icon>
      </span>
      <span
        class="mat-button-ripple mat-ripple mat-button-ripple-round"
        matripple=""
      />
      <span
        class="mat-button-focus-overlay"
      />
    </button>
  </mat-toolbar>
</app-header>
`;
